postman/collections/hearing_events.postman_collection.json

```diff
@@ -0,0 +1,80 @@
+{
+	"info": {
+		"_postman_id": "eb8fae8c-8499-40e5-99b0-5e8ecce810e1",
+		"name": "Hearing Events",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "Get Hearing Event logs on Date",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"pm.test(\"Status code is 200\", function () {",
+							"    pm.response.to.have.status(200);",
+							"});",
+							"",
+							"pm.test(\"Hearing Events Included\", function () {",
+							"    var json_data = pm.response.json();",
+							"    pm.expect(json_data).to.haveOwnProperty(\"hearing_id\")",
+							"    pm.expect(json_data).to.haveOwnProperty(\"has_active_hearing\")",
+							"    pm.expect(json_data).to.haveOwnProperty(\"events\")",
+							"});",
+							"",
+							"pm.test(\"Hearing Events is Array\", function () {",
+							"\tvar json_data = pm.response.json();",
+							"\tvar hearing_event = json_data.events;",
+							"\tpm.expect(hearing_event).to.be.a('array');",
+							"});",
+							"",
+							"pm.test(\"Hearing Events Properties Are Correct\", function () {",
+							"\tvar json_data = pm.response.json();",
+							"\tvar hearing_event = json_data.events[0];",
+							"\tpm.expect(hearing_event).to.haveOwnProperty('id');",
+							"\tpm.expect(hearing_event).to.haveOwnProperty('definition_id');",
+							"\tpm.expect(hearing_event).to.haveOwnProperty('defence_counsel_id');",
+							"\tpm.expect(hearing_event).to.haveOwnProperty('recorded_label');",
+							"\tpm.expect(hearing_event).to.haveOwnProperty('event_time');",
+							"\tpm.expect(hearing_event).to.haveOwnProperty('last_modified_time');",
+							"\tpm.expect(hearing_event).to.haveOwnProperty('alterable');",
+							"\tpm.expect(hearing_event).to.haveOwnProperty('note');",
+							"});",
+							""
+						],
+						"type": "text/javascript"
+					}
+				}
```

Comment:
This test fails on valid scenarios (Event Log has no events), therefore is not needed. 
This extra level of detail can be considered in contract testing as opposed to API testing

Reply:
I've removed this extra level of testing 👍🏾